app.js

```diff
@@ -37,6 +37,7 @@ app.io = io;
 // view engine setup
 app.set("views", path.join(__dirname, "views"));
 app.set("view engine", "jade");
+app.locals.moment = require('moment');
```

Comment:
As you can see in the screenshot, it shows `about 40 minutes ago` This is achieved using momentjs